client/src/components/chessboard/chessboard.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Chess from 'chess.js';
+import './chessboard.css';
+
+const chess = new Chess();
+const squares = chess.SQUARES;
```

Comment:
Comment out here the type of squares so that the user can know what squares consist of.